sqlglot/parser.py

```diff
@@ -1708,6 +1709,9 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
             elif create_token.token_type == TokenType.VIEW:
                 if self._match_text_seq("WITH", "NO", "SCHEMA", "BINDING"):
                     no_schema_binding = True
+            elif create_token.token_type == TokenType.WAREHOUSE:
+                this = self._parse_warehouse(identifier=table_parts.this)
+                extend_props(self._parse_properties())
```

Comment:
We don't need this

Reply:
This should be reflected in latest commit.